docs/release/release_0_4_14.md

```diff
@@ -0,0 +1,68 @@
+# napari 0.4.14
+
+We're happy to announce the release of napari 0.4.14!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+This is a small release containing some new features and fixing a few issues that have come up since `0.4.13` was released.
+
+This release adds the ability to convert shapes layers to labels layers in a context menu available on right clicking the layer (#3978). Our vectors layer has gained the 
+ability to render vectors in 2D across multiple slices (#2902) and points in the points
+layer can now be hidden individually (#3625).
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Complete list of changes below:
+
+## Highlights
+- Enable shapes layer (right click) > convert to labels (#3978)
+- out-of-slice rendering for Vectors (#2902)
+- Hide points individually (#3625)
+
+## New Features
+
+## Improvements
+- Update tests to remove warnings (#3974)
+- add test to check that things are removed properly (#3996)
+- Change base point slice thickness to half a unit (#3997)
+- Change base vector slice thickness to half a unit (#4001)
+
+## Bug Fixes
+- fix about on python 3.10 (#3972)
+- Revert changes to scikit-image test API (#3979)
+- Fix missing builtins in bundle (#3982)
+- don't use texture_format auto if float textures not available on machine (#3990)
+- Fix error when setting ndisplay=3 with empty shapes layer in viewer (#4003)
+- Fix bug preventing close button on dock widgets (#4006)
+
+## API Changes
+
+## Deprecations
```

Comment:
Just as a reminder, if #4007 gets in, then it probably contributes an API change (`out_of_slice_display` is a new param/property/event) and a deprecation (`n_dimensional` is deprecated).

Reply:
Note: API Changes are reserved for *existing APIs that we break*. *Addition* of parameters/attributes are not API changes.

Reply:
just to check @andy-sweet - `n_dimensional` is not deprecated now, right?

Reply:
It's deprecated/discouraged in the docstrings, but we don't issue deprecation warnings. So yeah, I think we should omit it here until those warnings are added.

Reply:
Perfect, thanks!